ink-wrapper/src/main.rs

```diff
@@ -46,15 +45,11 @@ fn main() -> Result<()> {
     let code_hash = metadata.source.hash;
     let metadata: InkProject = serde_json::from_str(&jsonized)?;
 
-    let tokens: rust::Tokens = generate(&metadata, code_hash, args.wasm_path);
+    let tokens: proc_macro2::TokenStream = generate(&metadata, code_hash, args.wasm_path);
 
     let stdout = std::io::stdout();
-    let mut w = fmt::IoWriter::new(stdout.lock());
 
-    let fmt = fmt::Config::from_lang::<Rust>().with_indentation(fmt::Indentation::Space(4));
+    stdout.lock().write_all(tokens.to_string().as_bytes())?;
```

Comment:
I've simplified the approach here since the `quote` output didn't seem to be compatible with it anymore.